src/vivarium/framework/population/manager.py

```diff
@@ -216,12 +223,6 @@ def get_view(
         The requested population view can be used to view the current state or
         to update the state with new values.
 
-        If the column 'tracked' is not specified in the ``columns`` argument,
```

Comment:
This is all handled differently now. 

1. the check for the "tracked" column happens in PopulationView.get() when it calls the PopulationManager.get_population().
2. We don't muck around with the query string anymore. Instead, if "tracked" is not in the pop view's columns, then the get_population() method simply removes the tracked simulants directly (prior to the query getting applied)